docs/designers-developers/developers/themes/theme-support.md

```diff
@@ -388,9 +388,9 @@ To make the content resize and keep its aspect ratio, the `<body>` element needs
 add_theme_support( 'responsive-embeds' );
 ```
 
-## Cover block padding
+## Spacing control
 
-Some blocks can provide padding controls in the editor for users. This is off by default, and requires the theme to opt in by declaring support:
+Using the Gutenberg plugin (version 8.3 or later), some blocks can provide padding controls in the editor for users. This is off by default, and requires the theme to opt in by declaring support:
 
 ```php
 add_theme_support('custom-spacing');
```

Comment:
This makes me wonder why the  Gutenberg plugin is needed? Did we just miss backporting this theme support flag?

Reply:
Not sure about why (the timing, another context that I'm missing) but we didn't port it https://github.com/WordPress/wordpress-develop/blob/master/src/wp-admin/edit-form-blocks.php#L144

Reply:
Ok, can we create a trac ticket for that and targetted for 5.7?

Reply:
https://core.trac.wordpress.org/ticket/51760#ticket I don't think I can assign the 5.7 milestone but marked the version as `trunk`.